eslint-warning-thresholds.json

```diff
@@ -8,9 +8,6 @@
   "packages/design-system-react-native/src/components/BadgeIcon/BadgeIcon.test.tsx": {
     "jest/prefer-strict-equal": 1
   },
```

Comment:
Eslint violation json updated to remove current violations after running `yarn lint`